shared/api/network.go

```diff
@@ -38,6 +38,10 @@ type Network struct {
 
 	// API extension: network
 	Managed bool `json:"managed" yaml:"managed"`
+
+	// API extension: clustering
+	State string   `json:"state" yaml:"state"`
```

Comment:
Should be Status (and "status" in API) for consistency

Reply:
Containers have always had a status field as do operations. Both of those use Status and StatusCode for that.

Reply:
This was fixed in 73d0351a1ce6716830848edb68499ef23b61c41c.